internal/rbd/controllerserver.go

```diff
@@ -274,7 +270,8 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 				return nil, err
 			}
 		}
-		return buildCreateVolumeResponse(ctx, req, rbdVol)
+
```

Comment:
@nixpanic is this new line intentionally kept?

Reply:
Yes. After a closing `}`, if the next statement is not related to the previous code-block, I prefer an empty line to split the parts of the code.

Reply:
Ok.. Sure..